Steer/DigitizerWorkflow/src/SimReaderSpec.cxx

```diff
@@ -143,6 +197,10 @@ DataProcessorSpec getSimReaderSpec(SubspecRange range, const std::vector<std::st
 
       mgr.getInteractionSampler().init();
       mgr.getInteractionSampler().print();
+
+      // update local GRPLHCIF with bunch filling schema
+      updateGRPLHCIF(o2::base::NameConf::getGRPLHCIFFileName(simprefixes[0]), mgr.getInteractionSampler().getBunchFilling());
```

Comment:
what speaks against doing this also in the GRPUpdater device? Might not need the code duplication.

Reply:
GRPUpdater collects inputs from all digitizer devices and updates the GRPECS (and still old monolithic GRP also). `updateGRPLHCIF` just adds the bunchFilling to GRPLHCIF, this does not depend on other detectors / digitizers.
Which code is duplicated (apart from the interprocess semaphor which is already duplicated in a few places, perhaps it should be extracted to separate utility)?

Reply:
ok, can change it bit later, now on other branch. In any case, I will need https://github.com/AliceO2Group/AliceO2/pull/8708 merged 1st.

Reply:
@shahor02 : I think we can merge #8708 only after we have default CCDB objects for everything (which might take a while). But I can rework the other PR also after this PR has been merged.

Reply:
Isn't it only imposing default runNumber 300000 in sim_challenge and full_system_test which requires a retrofitted CCDB?
If we would set a the moment some pilot beam run by default (and make it settable via an env.var) it should work, right?
I just want to have correct values set to the GRPs, i.e. the runNumber (via HBFUtils) and run start.

Reply:
True. We could set a different default run number for sim.

Reply:
@sawenzel moved GRPLHCIF update to GRPUpdater. While doing this, got a question: why do we send tens of copies of the same collision context with different subspecs,  
https://github.com/AliceO2Group/AliceO2/blob/3c5523758f34bf9c50a640348c900fabf79e312a/Steer/DigitizerWorkflow/src/SimReaderSpec.cxx#L209-L216 ?

Will wait for https://github.com/AliceO2Group/AliceO2/pull/8708 to set correct runNumber and time.